### PR TITLE
add homework databases-week1

### PR DESCRIPTION
**Pull request by TatjanaOcedova**
Add homework databases-week1

### CODE REVIEW

File: databases/week1/homework.sql [hunk 1]
```diff
@@ -0,0 +1,103 @@
+Select
+*
+FROM
+task;
+
+-- 1.Find out how many tasks are in the task table
+SELECT COUNT(id)
```
Comment (lawansubba) [inline databases/week1/homework.sql]: COUNT(task.id) as task_Count
_Pointer_
Add column name for columns you have created

File: databases/week1/homework.sql [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+SELECT
+    *
+FROM
+    task
+JOIN
+    status
+ON
+    task.status_id = status.id
+WHERE 
+    name LIKE "Done";
```
Comment (lawansubba) [inline databases/week1/homework.sql]: WHERE
    status.name = "Done";
_Pointer_
Use Like operator for only pattern matches, use of Equals operator is more appropriate

File: databases/week1/homework.sql [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+FROM
+    status;
+SELECT
+    *
+FROM
+    task
+JOIN
+    status
+ON task.status_id = status.id
+WHERE name LIKE "Not started" OR name LIKE "In progres";
```
Comment (lawansubba) [inline databases/week1/homework.sql]: WHERE
    status.name != "Done";
_Pointer_
Use Like operator for only pattern matches, use of not Equals operator is more appropriate

File: databases/week1/homework.sql [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+SELECT
+    title, due_date
+FROM
+    task
+WHERE
+    title LIKE "%database%" OR description LIKE "%database%";
+
+-- 8.Get the title and status (as text) of all tasks
+SELECT
+    title, name
```
Comment (lawansubba) [inline databases/week1/homework.sql]: task.title, status.name
_pointer_
Use tablename.column for readability and to remove ambiguity when joins are done

File: databases/week1/homework.sql [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+FROM
+    task
+JOIN
+    status 
+ON task.status_id = status.id;
+
+-- 9.Get the name of each status, along with a count of how many tasks have that status
+SELECT 
+    name,
+    COUNT(title)
```
Comment (lawansubba) [inline databases/week1/homework.sql]: status.name
COUNT(task.title) as Title_Count
_Pointer_
1) Give name to columns that you have created
2) Use tablename.column for readability and to remove ambiguity when joins are done

**Review by lawansubba — COMMENTED** [review]
Good work!!  please address the minor comments

**Comment by TatjanaOcedova** [comment]
Thank you for the review 😊

**Pull request merged**